packages/server/database/migrations/20200828161500-samlDomainAsId.ts

```diff
@@ -0,0 +1,26 @@
+import {R} from 'rethinkdb-ts'
+export const up = async function(r: R) {
+  try {
+    await r
+      .table('SAML')
+      .update((row) => ({
+        id: row('domain')
```

Comment:
did this work? 
this should produce an error, the primary key cannot be updated.
you'll need to clone the rows